### PR TITLE
[15.0][MIG] sale_financial_risk_info: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa
TT40268

### CODE REVIEW

**Comment by ernestotejeda** [comment]
ping @carlosdauden @pedrobaeza

File: sale_financial_risk_info/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Sale Financial Risk Info",
     "summary": "Adds risk consumption info in sales orders.",
-    "version": "14.0.1.1.0",
+    "version": "15.0.1.1.0",
```
Comment (pedrobaeza) [inline sale_financial_risk_info/__manifest__.py]:
```suggestion
   "version": "15.0.1.0.0",
```

File: sale_financial_risk_info/models/sale.py [hunk 2]
```diff
@@ -9,10 +9,6 @@ class SaleOrder(models.Model):
     _inherit = "sale.order"
 
     risk_info = fields.Html(compute="_compute_risk_info")
-    risk_remaining_value = fields.Monetary(related="partner_id.risk_remaining_value")
```
Comment (pedrobaeza) [inline sale_financial_risk_info/models/sale.py]: This shouldn't be removed. You didn't rebase properly.

File: sale_financial_risk_info/tests/test_sale_financial_risk_info.py [hunk 3]
```diff
@@ -22,7 +22,7 @@ def setUp(self):
         )
         self.default_account_revenue = self.env["account.account"].search(
             [
-                ("company_id", "=", self.env.user.company_ids[0].id),
+                ("company_id", "=", self.env.user.company_id.id),
```
Comment (pedrobaeza) [inline sale_financial_risk_info/tests/test_sale_financial_risk_info.py]:
```suggestion
                ("company_id", "=", self.env.company.id),
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: sale_financial_risk_info/models/sale.py (hunk 2 above)
Comment (ernestotejeda) [reply sale_financial_risk_info/models/sale.py]: I have rebased and I can't see in 15.0 the `risk_remaining_value` field for the res.partner model in the `sale_financial_risk` module and therefore I am getting an error having a related partner_id.risk_remaining_value field in sale.order in this (sale_financial_risk_**info**) module. But also not even in v14 that related is used in this module (nothing is shown or done with it)

Comment (pedrobaeza) [reply sale_financial_risk_info/models/sale.py]: This module is not in v15. And in v14, it's here: https://github.com/OCA/credit-control/blob/771640902939557782bd87a4a6efacd0d6fc5351/sale_financial_risk_info/models/sale.py#L12

The problem is that your migration commit is removing these fields.

Comment (ernestotejeda) [reply sale_financial_risk_info/models/sale.py]: I know what happens. I did research and the PR that adds the field was done in v14.0 after the migration to v15 of the account_financial_risk module, I did a forward-port of that PR to recover those fields in res.partner. 
PR: https://github.com/OCA/credit-control/pull/239

**Comment by pedrobaeza** [comment]
Please rebase and redo the migration without the unwanted changes.

**Comment by ernestotejeda** [comment]
@pedrobaeza rebased and undone that change

**Review by pedrobaeza — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**